tests/test_core.py

```diff
@@ -6705,7 +6710,7 @@ class std_string {
   std_string(const std_string& s): ptr(s.ptr) { std::cout << "std_string(const std_string& s) " << std::endl; }
   const char* data() const { return ptr; }
 private:
-  const char* ptr = nullptr;
```

Comment:
This removes warnings? I'd have thought `nullptr` was the more-correct form.

Reply:
but we don't have c++11 by default so its a warning.